plugins/org.jkiss.dbeaver.ext.generic/src/org/jkiss/dbeaver/ext/generic/model/GenericExecutionContext.java

```diff
@@ -387,4 +398,12 @@ public GenericObjectContainer getDefaultObject() {
         return null;
     }
 
+    @NotNull
+    @Override
+    public DBCCachedContextDefaults getCachedDefault() {
```

Comment:
You cannot call getDefaultSchema or getDefaultCachedCatalog here. 
you cannot interact with dynamic catalog or schema.
Use just selectedEntityName